### ISSUE TITLE
AFNetworking rather than ASIHTTPRequest

### ISSUE THREAD

**Issue by leehambley** [ChristianKienle/OSXBoilerplate#3]
I'm new to developing for OSX and iOS, but it seems that AFNetworking is the current best of breed for networking code, and that ASIHTTPRequest has lost favor owing to being too complicated and abandoned by its' author.

**Comment by ChristianKienle** [comment]
You are absolutely correct. Thats why I have removed ASIHTTPRequest months ago. Its only mentioned in the documentation because I had no time to remove it from there. When creation OS X Boilerplate AFNetworking wasn't ported to the Mac. I started to port it - published a working port but did not continue. I will play around with AFNetworking a bit to see if its stable (the Mac Port) and then integrate it. If you want to send me a push request: you are welcome. :)

**Comment by leehambley** [comment]
@ChristianKienle please excuse me for not looking properly at the code.

As a beginner writing OS X Applications I will live or die based on the documentation. I'm quite sure that a pull request from me wouldn't bring anything, as I'm not qualified (not even close!) to decide if something is prime for inclusion in a project boilerplate, let alone a production application.

FWIW the _AFNetowrking_ team appear to have ported it themselves, their [project description](https://github.com/AFNetworking/AFNetworking) reads:

> AFNetworking is a delightful networking library for iOS and Mac OS X. It's built on top of NSURLConnection, NSOperation, and other familiar Foundation technologies.

**Comment by ChristianKienle** [comment]
@leehambley You don't have to excuse yourself. I would like to thank you for finding a flaw in the documentation and for telling me about the Mac Port of AFNetworking. If all goes well I will have time in a few days to fix this issue. Until then I keep this issue open. Thanks again for reporting it! Welcome as a OS X developer. You will enjoy it. :-)

**Comment by leehambley** [comment]
@ChristianKienle thanks for the kind words. I'll probably be back around in a few weeks, ironically my first OSX project requires that I build a web app first (at least is't playing to my strengths) so that the team can build the initial database!

**Comment by ChristianKienle** [comment]
@leehambley You are from Hamburg. Will you be at the Macoun conference in October? If you wanna have a chat send me your IM-details please. (me@christian-kienle.de) :) Would love to see you there.

**Comment by leehambley** [comment]
@ChristianKienle I hadn't thought about it, but maybe I should register, I've never been to Frankfurt! Thanks for the tip. (See you email for IM info)
